.github/workflows/build.yml

```
@@ -198,6 +198,51 @@ jobs:
             artifacts/**/*.tar.gz
             artifacts/**/*.zip
 
+  docker:
+    needs: [ unit-test, versioning ]
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)